### PR TITLE
Web app manifest (fixes #9)

### LINKED ISSUE #9: Add manifest.json
A manifest.json would make adding this site to a mobile phone home screen more seamless, so it opens independently as a mini kiosk app rather than simply opening in the Chrome browser.

A simple icon might be based on the button design?

### PR DESCRIPTION
**Pull request by Rudloff**
Hello,

This patch adds a web app manifest, that allows to easily add the app to the Android homescreen.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+    "name": "Are men talking too much?",
+    "description": "Check who's dominating the conversation",
+    "start_url": "./",
+    "backgorund-color": "#FFFFFF",
```
Comment (MatthewWilkes) [inline manifest.json]: Looks like a typo.

Comment (Rudloff) [inline manifest.json]: You're right. I updated my PR.

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+    "name": "Are men talking too much?",
+    "description": "Check who's dominating the conversation",
+    "start_url": "./",
+    "background-color": "#FFFFFF",
```
Comment (cathydeng) [inline manifest.json]: shouldn't this be `background_color`? is there a good way to validate a manifest.json file for a website?

Comment (Rudloff) [inline manifest.json]: Yes, sorry for the typo.
I just found this validator: https://manifest-validator.appspot.com/

**Pull request merged**